conda_smithy/configure_feedstock.py

```diff
@@ -1060,6 +1060,7 @@ def _appveyor_specific_setup(jinja_env, forge_config, forge_dir, platform):
     build_setup = new_build_setup.strip()
 
     forge_config["build_setup"] = build_setup
+    forge_config["appveyor"] = {"image": "Visual Studio 2017"}
```

Comment:
Can this use the `_load_forge_config` function where this can be overriden?

Reply:
IDK where that is, but yeah maybe. I'll look.

Reply:
done.